018_go_list_mod_graph_why/README.md

```diff
@@ -0,0 +1,459 @@
+<!-- __JSON: gobin -m -run myitcv.io/cmd/egrunner script.sh # LONG ONLINE
+
+### Introduction
+
+`go list`, `go mod why` and `go mod graph` can be used to analyse modules and packages. In this guide we introduce each
+command as we analyse the `{{PrintOut "main module"}}` module:
+
+{{PrintOut "base repo"}}
+
+For any of the commands, see `go help` for more details, for example, `go help list` or `go help mod graph`. These help
```

Comment:
This sentence reads weird to me. Perhaps use a period before "for example"?

Reply:
Done.